src/nplinker/scoring/linking/link_finder.py

```diff
@@ -442,8 +446,8 @@ def get_links(self,
             link_levels = [0, 1]
 
             # Get necessary ids
-            # CG: TODO update the logics here
-            #   integer gcf.id has been removed, use string gcf.gcf_id instead.
+            # CG: TODO update the logics here:
+            #   don't use integer gcf.id, use string gcf.gcf_id instead.
             input_ids = np.array([gcf.id for gcf in input_object],
                                  dtype=np.int32)
```

Comment:
Should this todo be adressed immediately or do we plan to take care of this later?

Reply:
Thought about that, better later to fix it.